gqueries/output_elements/output_series/mekko_of_kerosene_demand_supply/kerosene_demand_bio_industry.gql

```diff
@@ -1,2 +1,6 @@
 - unit = MJ
-- query = V(industry_final_demand_bio_kerosene, input_of_bio_kerosene)
+- query = 
+      SUM(
+        V(industry_final_demand_bio_kerosene, input_of_bio_kerosene),
+        V(industry_final_demand_bio_kerosene_non_energetic, input_of_bio_kerosene)
```

Comment:
Can't we make these queries a bit more robust? Or even better, can't we just refer to some standard query for this in the general folder?